CONTRIBUTING.md

```diff
@@ -54,3 +54,28 @@ $ git tag -a -s -m vX.Y.Z vX.Y.Z <release-commit-ref>
 
 For this step, it is necessary to [configure Git to sign tags with SSH
 keys](https://docs.gitlab.com/ee/user/project/repository/signed_commits/ssh.html).
+
+## Pull Request Guidelines
+
+Ensure that each proposed commit is an operation that applies a set of distinct
+changes as a single operation (in other words, an atomic commit). The title
+should follow the pattern `<scope>: <description>`. The message should properly
```

Comment:
Shouldn't we add a note that the `scope` should be one of the previously used scopes if possible?